omc3/plotting/plot_checked_corrections.py

```diff
@@ -403,6 +429,72 @@ def save_plots(output_dir, figure_dict, input_dir=None):
             fig.savefig(output_path)
 
 
+def show_plots(figure_dict: Dict[str, Figure]):
+    """ Shows plots.
+    If qtpy is installed, they are shown in a single window.
+    The individual corrections are sorted in to vertical tabs,
+    the optics parameter into horizontal tabs. 
+    If qtpy is not installed, they are simply shown as individual figures.
+    This is not recommended
+    """
+    try:
+        window = VerticalTabWindow("Correction Check")
+    except TypeError:
+        log_no_qtpy_many_windows()
+        for name, fig in figure_dict.items():
+            create_pyplot_window_from_fig(fig, title=name.replace(SPLIT_ID, " "))
+        plt.show()
+        return
+
+    rdt_pattern = re.compile(r"f\d{4}")
+    rdt_complement = {
+        RDT_REAL_COLUMN.text_label: RDT_IMAG_COLUMN,
+        RDT_AMPLITUDE_COLUMN.text_label: RDT_PHASE_COLUMN,
+    }
+
+    correction_names = sorted(set([n.split(SPLIT_ID)[0] for n in figure_dict.keys() if SPLIT_ID in n]))
```

Comment:
Here n is the name, right? Let's rename accordingly

Reply:
n is more of an id than a name

Reply:
As I hold the belief that k is a far superior letter to n, I'm fine with the change